sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -34,7 +34,7 @@ protected FormTrainingClient()
         /// <summary>
         /// Initializes a new instance of the <see cref="FormTrainingClient"/> class.
         /// </summary>
-        /// <param name="endpoint">The endpoint to use for connecting to the Form Recognizer Azure Cognitive Service. The URI is likely to be similar to <c>{protocol}://{resourcename}.cognitiveservices.azure.com</c>.</param>
+        /// <param name="endpoint">The endpoint to use for connecting to the Form Recognizer Azure Cognitive Service.</param>
```

Comment:
We removed the last part of the description in the `FormRecognizerClient` constructors in last preview, but forgot to remove from the `FormTrainingClient`.